install/disconnected_install.adoc

```diff
@@ -36,35 +37,24 @@ and plan your environment topology.
 with access to the Internet and at least 110 GB of disk space. You download the
 required software repositories and container images to this computer.
 
-* Plan to add a webserver within your disconnected environment to serve the software and
-required images. If your organization's policies allow you to move the RHEL 7
-server that you download the required software repositories and container images
-to into your disconnected LAN, you can plan to reuse that server. If you cannot
-move the server into your disconnected LAN, then you create a separate webserver
-during installation preparation.
+* Plan to maintain a webserver within your disconnected environment to serve the
+mirrored repositories. You copy the repositories from the Internet-connected
+host to this webserver, either over the network or by using physical media in disconnected
+deployments.
 
 * Provide a source control repository. After installation, your nodes must
 access source code in a source code repository, such as
 Git.
-
++
 When building applications in {product-title}, your build might contain
 external dependencies, such as a Maven Repository or Gem files for Ruby
 applications.
 
-[NOTE]
-====
-You can use a
-http://www.redhat.com/en/technologies/linux-platforms/satellite[Red Hat
-Satellite] server to provide access to Red Hat content via an intranet or
-LAN. For environments with Satellite, you can synchronize the {product-title}
-software onto the Satellite for use with the {product-title} servers.
-
-https://access.redhat.com/documentation/en/red-hat-satellite/[Red Hat Satellite
-6.1] also introduces the ability to act as a Docker registry, and it can be used
-to host the {product-title} containerized components. Doing so is outside of the
-scope of this document.
-====
-
+* Provide a registry within the disconnected environment. Options include:
+** Installing a
+xref:../install/stand_alone_registry.adoc#install-config-installing-stand-alone-registry[stand alone {product-title} registry].
+** Using a https://access.redhat.com/documentation/en/red-hat-satellite/[Red Hat Satellite
+6.1] server that acts as a Docker registry.
```

Comment:
@kalexand-rh I am ignorant of Satellite. Is setting up Satellite as a _Docker registry_ something specific? Or, just a standard registry to hold Docker images?

Reply:
@mburke5678, all I know is that in that version and later you can tick a box to make it work as a Docker registry. I don't know more about Satellite than you do.